docs/modules/operation/pages/events/sources/send-event.adoc

```diff
@@ -15,7 +27,7 @@ For example:
 
 Format: `bin/send-event.pl <uei> [host:port] [options]`
 
-The `host:port` defines the hostname and port to send the event; by default the local host.
+The `host:port` defines the hostname and port to send the event; by default, the local host.
```

Comment:
The last part of this sentence ("by default, the local host") confuses me. Maybe we should provide more detail?

Reply:
@mmahacek , do you have suggestions for how to update this beyond the change I made?

Reply:
I don't know that I have any other suggestions on this line.  By default, the destination is the system where the command is run, aka "localhost"